relay-event-normalization/src/normalize/mod.rs

```diff
@@ -30,14 +30,14 @@ use crate::{
 
 pub mod breakdowns;
 pub mod nel;
+pub(crate) mod processor;
```

Comment:
Why does it have to be `pub(crate)` ?

Reply:
I'm ok with this if this will be revisited in the followup.  If the module made `pub(crate)` it means the structure of the modules is not really optimal and some code should be moved around. Like tests in this case.

Reply:
I agree with the suboptimal module structure. I'm going to merge this as-is and address it in a follow-up PR.

Reply:
Resolving as this is addressed in a follow-up PR: https://github.com/getsentry/relay/pull/2729.